update marina doc + add ionio-example (#18)
vulpemventures/docs.vulpem.com: docs/marina/api.md

### createAccount

```typescript
marina.createAccount(accountID: AccountID, accountType: AccountType): Promise<void>;
```

Open a popup, ask the password locking the marina private key.
If the user accepts, marina will create a new account.

Account types are:

>  * `AccountType.P2WPKH`: a standard segwit v0 account (default)

Reviewer: ```suggestion
 * `AccountType.P2WPKH`: native segwit v0 account (default)
```

>  * `AccountType.Ionio`: an account using [Ionio](https://ionio-lang.org/) Artifact to generate taproot script.

Reviewer: ```suggestion
 * `AccountType.Ionio`: account using [Ionio](https://ionio-lang.org/) artifacts to generate taproot scripts.
```

:::tip

> All accounts are *confidential* and generates blinding keys for each script according to [SLIP77](https://github.com/satoshilabs/slips/blob/master/slip-0077.md).

Reviewer: ```suggestion
All accounts are *confidential* and generate blinding keys for each script according to [SLIP77](https://github.com/satoshilabs/slips/blob/master/slip-0077.md).
```

:::

### getAccountInfo

```typescript
marina.getAccountInfo(accountID: AccountID): Promise<AccountInfo>;
```
Get the account info of the account `accountID` if it exists. Returns an error otherwise.

```typescript
export interface AccountInfo {
  accountID: AccountID;
  type: AccountType; 
  masterXPub: string;
  baseDerivationPath: string; // path to masterXPub
  accountNetworks: NetworkString[]; // the account is available on these networks
}
```

### useAccount

```typescript
marina.useAccount(accountID: AccountID): Promise<void>;
```
Switch to account with ID = `accountID` if it exists. By default, current selected account is always "mainAccount".

### isEnabled

```typescript
marina.isEnabled(): Promise<boolean>
```

Returns whether the user has already granted permission to the website to access their wallet.
If this returns false, some methods will throw an error.

### isReady

```typescript
marina.isReady(): Promise<boolean>

> ``` 

Reviewer: remove trailing space
```suggestion
```
```

> Returns true if the user has already set up a wallet.

Reviewer: ```suggestion
Returns whether the user has already set up a wallet.
```

### enable

```typescript
marina.enable(): Promise<void>
```

> Ask the user to grant permission to the website to access their wallet. It will open a popup and wait for the user response.

Reviewer: ```suggestion
Ask the user to grant permission to the website to access his wallet. It will open a popup and wait for the user's response.
```

### disable

```typescript
marina.disable(): Promise<void>
```

> Disable the website to access the user's wallet. It does not open any popup and does not need user interaction.

Reviewer: ```suggestion
Deny the website to access the user's wallet. It does not open any popup and does not need user interaction.
```

### getNetwork

```typescript
marina.getNetwork(): Promise<'liquid' | 'testnet' | 'regtest'>
```

> Returns the network the wallet is connected to.

Reviewer: ```suggestion
Returns the network to which the wallet is connected.
```
